lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -5,9 +5,16 @@
 from django.utils.translation import pgettext
 from openedx.core.djangolib.markup import HTML, Text
 %>
-% if section_data['access']['admin']:
 <fieldset class="batch-enrollment membership-section">
     <legend id="heading-batch-enrollment" class="hd hd-3">${_("Batch Enrollment")}</legend>
+    % if not section_data['access']['admin']:
+    <label>
+      ${_("Batch Enrollment is disabled. Please ask users to enroll via ")}
+      <a href="https://mitxonline.mit.edu">mitxonline.mit.edu</a>
```

Comment:
Is there some way to pull this link from the environment so that it will work on RC and production? 

If it takes more than 5 minutes effort, it's not worth doing.

Reply:
> Is there some way to pull this link from the environment so that it will work on RC and production?
> 
> If it takes more than 5 minutes effort, it's not worth doing.

It was already in the environment. I just added it now. Sorry for missing it before